### PR TITLE
docs(portal): add component documentation

### PR DESCRIPTION
**Pull request by KekmaTime**
Closes https://github.com/yamada-ui/yamada-ui/issues/1220 

## Description

JSDoc is  missing from the `Portal` component.

## New behavior

I have added JSDoc for the `Portal` component.

## Is this a breaking change (Yes/No):

No

### CODE REVIEW

**Comment by hirotomoyamada** [comment]
Please run `pnpm changeset` and select a `patch`.

**Comment by hirotomoyamada** [comment]
@KekmaTime 

Hi, thanks for the PR.

Pls run `pnpm changeset` and select a `patch`.

File: package.json
Comment (setodeve) [inline package.json:1]: Please remove this file from PR.

File: pnpm-lock.yaml
Comment (setodeve) [inline pnpm-lock.yaml:1]: Please remove this file from PR.

**Review by setodeve — CHANGES_REQUESTED** [review]

**Comment by hirotomoyamada** [comment]
@KekmaTime 

Rather than deleting the file, it removes it from the commit.

File: package.json
Comment (setodeve) [reply package.json:1]: @KekmaTime 
Sorry about that.
Please do not include changes to `package.json` in this PR.

File: pnpm-lock.yaml
Comment (setodeve) [reply pnpm-lock.yaml:1]: @KekmaTime 
Please do not include changes to `pnpm-lock.yaml` in this PR.

**Pull request closed without merging**